stream_chat/base/client.py

```diff
@@ -384,6 +384,12 @@ def unflag_user(
         """
         pass
 
+    @abc.abstractmethod
+    def mute_users(
+        self, target_ids: List[str], user_id: str, **options: Any
```

Comment:
I suppose we can remove `user_id` as required argument, the same for `unmute`.
```suggestion
        self, target_ids: List[str], **options: Any
```

It is required when request can be done only server-side, and should be optional if it can be done from client- and server-side.

Reply:
Well, tbh, the mute_users method is nothing more than a wrapper for the mute_user method, which also accepts `user_id`. I wouldn't change this.

Reply:
I agree that consistency is good. Still, just to be clear: `user_id` is not required for all API calls, but is required as an argument.

Maybe sometime later `user_id` can be removed from all calls that don't require it. 👍